### ISSUE TITLE
Build is failing when I try to build for iOS with ssc built with `publish-npm-packages.sh --link`

### ISSUE THREAD

**Issue by chicoxyzzy** [socketsupply/socket#758]
```
➜ ssc build --platform=ios -c -p && ssc install-app --platform=ios
warn - using '/usr/bin/clang++' as CXX
• cleaned: /Users/chicoxyzzy/dev/socketsupply/relay/dist/ios +134ms
• ERROR: libs folder for the target platform doesn't exist: /Users/chicoxyzzy/dev/socketsupply/socket/build/npm/darwin/packages/@socketsupply/socket-darwin-x64/lib/arm64-iPhoneOS +81ms
• ERROR: objects folder for the target platform doesn't exist: /Users/chicoxyzzy/dev/socketsupply/socket/build/npm/darwin/packages/@socketsupply/socket-darwin-x64/objects/arm64-iPhoneOS +0ms
```

This error doesn't appear when I use ssc which is built with `install.sh`

**Comment by jwerle** [comment]
Closed with #762

**Comment by chicoxyzzy** [comment]
fixed in #762
